src/recipes/trash-can.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "Tash Can",
+  "description": "Tasty Mix of Whatever you Have on Hand with a Caffeine Boost!",
+  "github": "drahenshaw",
+  "ingredients": [
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Gin"
+    },
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Vodka"
+    },
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Triple Sec"
+    },
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Rum (Light)"
+    },
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Blue Caracao"
+    },
+    {
+      "quantity": "1/2",
+      "measure": "oz",
+      "ingredient": "Peach Schnapps"
+    },
+    {
+      "quantity": "1",
+      "measure": "can",
+      "ingredient": "Red Bull"
+    }
+  ],
+  "directions": [
+    "Fill large glass with ice. Add alcohol ingrediants. Open the Red Bull and place upside down in the glass."
+  ],
+  "image": "vodka-soda.jpg",
```

```suggestion
  "image": "trash-can.jpg",
```
Also, update the image filename to `trash-can.jpg` too. Thanks!

---

Oops, sorry about that! It should be resolved now! Thank you!